test/cli_test.go

```diff
@@ -288,7 +305,7 @@ func terraformInit(t *testing.T, executable, dir string) {
 }
 
 func terraformPlan(t *testing.T, executable, dir, tfplan string) {
-	terraformExec(t, executable, dir, "plan", "-input=false", "--out", tfplan)
+	terraformExec(t, executable, dir, "plan", "-input=false", "-refresh=false", "-out", tfplan)
```

Comment:
It looks like this is the change that's causing some IAM tests to fail on terraform 0.12. So the good news is that there's not a breaking change in the codebase, just in the tests. However, I will need to figure out a way to get the tests to pass on terraform 0.12 as well :-) This change is required in order to write update tests.

Reply:
It looks like the failing tests are all using an iam policy data source as an input for a different iam policy. Something in how that's handled (with respect to the -refresh option) must have changed between terraform 0.12 and 0.13. I don't think our tests need to use data sources; I'll remove them and enter the JSON values for the policies directly.